npm/src/lib/libDef.js

```diff
@@ -346,3 +354,48 @@ export async function getShallowGHLibDefs(
   }
   return _libdefs
 }
+
+const identity = i => i
+
+function getGHFlowVersionsForDef(def: ShallowGHLibDef): Promise<Array<Version>> {
+  const getContent = Rx.Observable.fromCallback(gitHubClient().repos.getContent)
+  const notNull = i => i != null
+  return getContent({
+    user: 'flowtype',
+    repo: 'flow-typed',
+    path: `/definitions/${def.libDef.pkgNameVersionStr}`
+  })
+  .flatMap(identity)
+  .filter(notNull)
+  .flatMap(identity)
+  .map(i => i.name)
+  .filter(name => name.indexOf('flow-') == 0)
+  .map(name => {
+    let withoutPrefix = name.match(/flow-(.*)/)[1]
```

Comment:
nit: might as well be `const`, eh?

(PS: "nit" === I don't feel strongly, feel free to object/ignore)